unicoremc/models.py

```diff
@@ -367,7 +432,8 @@ def create_pyramid_settings(self):
                 self.available_languages.all(),
                 self.default_language or Localisation._for('eng_GB'),
                 self.ga_profile_id,
-                self.hub_app()
+                self.hub_app(),
+                self.all_repos()[0].name()
```

Comment:
Shouldn't this be `self.own_repo` otherwise, for multi repos, this will a random name from any of the related repos.